src/main/java/org/jenkinsci/plugins/cloudstats/CloudStatistics.java

```diff
@@ -264,7 +274,7 @@ private XmlFile getConfigFile() {
     @Extension
     public static class ProvisioningListener extends CloudProvisioningListener {
 
-        private final CloudStatistics stats = CloudStatistics.get();
+        private final CloudStatistics stats = getCloudStatistics();
```

Comment:
This seems like a personal preference. Adding static method imports just to save one or couple of characters is not what I prefer. I am ok to add these to the top level class for convenience but without deprecating the old methods.